zerver/lib/user_groups.py

```diff
@@ -26,6 +32,34 @@ def access_user_group_by_id(user_group_id: int, user_profile: UserProfile) -> Us
     return user_group
 
 
+def get_user_group_from_ids(
+    user_group_ids: Sequence[int], acting_user: UserProfile
+) -> List[UserGroup]:
```

Comment:
The function name should be plural, and we probably want to call this `access_user_groups_by_id`.